test/strings/basic.jl

```diff
@@ -478,8 +478,8 @@ Base.endof(x::CharStr) = endof(x.chars)
 @test cmp(CharStr("\U1f596"), "\U1f596\U1f596") == -1
 
 # repeat function
-@test repeat("xx",3) == repeat("x",6) == "xxxxxx"
-@test repeat("αα",3) == repeat("α",6) == "αααααα"
+@test repeat("xx",3) == repeat("x",6) == repeat('x',6) == "xxxxxx"
+@test repeat("αα",3) == repeat("α",6) == repeat('α',6) == "αααααα"
```

Comment:
I would also add a regression test for `^` to make `'x'^0 == ""` and `'x'^1 == "x"`.